ARDemos/ARDemos/AppDelegate.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
```

Comment:
delete this please 😊

Reply:
Done 7a5d800e410935e5ceca5c8022281e81651de2a5